README.md

```diff
@@ -25,7 +25,47 @@ This README is meant as a brief walkthrough on the core features of CDC Connecto
 2. CDC connectors for DataStream API, users can consume changes on multiple databases and tables in a single job without Debezium and Kafka deployed.
 3. CDC connectors for Table/SQL API, users can use SQL DDL to create a CDC source to monitor changes on a single table.
 
-## Usage for Table/SQL API
+## Quick Start
+
+### Usage for Pipeline Connectors
+
+The example shows how to continuously synchronize data, including snapshot data and incremental data, from multiple business tables in MySQL database to Doris for creating the ODS layer.
+
+1. Download and extract the flink-cdc-3.0.tar file to a local directory.
+2. Download the required CDC Pipeline Connector JAR from Maven and place it in the lib directory (automated pulling using a script can be optimized in the future).
```

Comment:
Please delete `(automated pulling using a script can be optimized in the future)`